doc/client_lua_api.md

```diff
@@ -754,6 +759,22 @@ Call these functions only at load time!
     * returns reference to mod private `StorageRef`
     * must be called during mod load time
 
+### Mod channels
+You can find mod channels communication scheme in `docs/mod channels.png`.
```

Comment:
While the diagram isn't bad, mod channels should be explained in text too.

For example it isn't immediately obvious that clients can communicate with <b>other clients</b>.

Reply:
it's a string

Reply:
(still needs to be addressed)

Reply:
`![Mod channels communication scheme](docs/mod channels.png)`

also, use underscore rather than space